src/packages/@ncigdc/components/Aggregations/FacetResetButton.js

```diff
@@ -26,26 +28,31 @@ const StyledLink = styled(Link, {
   },
 });
 
-const FacetResetButton = ({ field, currentFilters, style, ...props }) => {
+const FacetResetButton = ({
+  field,
+  currentFilters,
+  query,
+  style,
+  ...props
+}) => {
   const newFilters = removeFilter(field, currentFilters);
-  const newQuery = newFilters
-    ? {
-        offset: 0,
-        filters: newFilters,
-      }
-    : {};
+  const newQuery = {
+    ...query,
+    offset: 0,
+    filters: newFilters,
+  };
   const inCurrent = fieldInCurrentFilters({
     currentFilters: currentFilters.content || [],
     field,
   });
   return (
     <StyledLink
       style={{ display: inCurrent ? 'inline' : 'none', ...style }}
-      query={inCurrent ? newQuery : {}}
+      query={newQuery}
```

Comment:
Can we just use this? Works and can understand what's going on without having to know the intricacies of how `StyledLink` works